src/AirshipBindings.iOS.Core/ApiDefinitions.cs

```diff
@@ -418,11 +418,6 @@ partial interface Constants
         [Obsolete("Deprecated – to be removed in SDK version 14.0. Please use UAFetchDeviceInfoActionDefaultRegistryName.")]
         NSString kUAFetchDeviceInfoActionDefaultRegistryName { get; }
 
-        // extern DEPRECATED_MSG_ATTRIBUTE( "Deprecated – to be removed in SDK version 14.0.") NSString *const kUAInteractionMCRAP
```

Comment:
This was causing linker errors for SampleApp.iOS, due to the issue fixed in https://github.com/urbanairship/ios-library-dev/pull/1975